Mage.Sets/src/mage/cards/c/CinderslashRavager.java

```diff
@@ -0,0 +1,58 @@
+package mage.cards.c;
+
+import java.util.UUID;
+import mage.MageInt;
+import mage.abilities.common.EntersBattlefieldTriggeredAbility;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.dynamicvalue.DynamicValue;
+import mage.abilities.dynamicvalue.common.PermanentsOnBattlefieldCount;
+import mage.abilities.effects.common.DamageAllEffect;
+import mage.abilities.effects.common.cost.SpellCostReductionForEachSourceEffect;
+import mage.constants.SubType;
+import mage.abilities.keyword.VigilanceAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Zone;
+import mage.counters.CounterType;
+import mage.filter.common.FilterControlledPermanent;
+import mage.filter.common.FilterOpponentsCreaturePermanent;
+
+/**
+ *
+ * @author AhmadYProjects
+ */
+public final class CinderslashRavager extends CardImpl {
+    private static final FilterControlledPermanent filter = new FilterControlledPermanent("permanents you control with an oil counter on it");
```

Comment:
Text should be "*permanent* you control with an oil counter on it" instead of "*permanents* you control with an oil counter on it"